### DISCUSSION TITLE
Out of video memory error

### DISCUSSION THREAD

**Discussion by TimeExplorerK** [carla-simulator/carla#7022] [General]
I'm new to using Carla, but when I try to start the program, I keep getting an error saying that my GPU is out of video memory, despite my GPU having 7.8 GBs of memory, and the website calling for only 6 GB (though 8 GB are recommended). I'm not sure what's wrong, do I really need a bigger GPU? I did try starting it using the low rendering mode as well.
Edit: Using an Intel(R) HD Graphics 520

**Comment by Dilcan44** [comment]
I had the exact same error, I tried updating my GPU NVIDIA drivers through the Geforce Experience app and it started working properly again.

**Reply by TimeExplorerK** [reply]
Hi, thanks for your response! I double checked the system requirements and found out I should be using a quad-core processor, which I'm guessing is the problem. My original processor was a dual-core, but I've since upgraded and had no issues!
